### PR TITLE
Moved benchmarks to separate module

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #106 
Moved benchmarks to separate module, moved dependencies and description from readme.

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Debian benchmarks
+
+Here is general algorithm to run benchmarks:
+ 1. Install snapshot locally of `debian-adapter`: `mvn install`
+ 2. Build `debian-bench` project: `mvn package -f ./benchmarks`
+ 3. Create directory and copy resources required for test into this directory
```
Comment (genryxy) [inline benchmarks/README.md:6]: @olenagerasimova I'd suggest to add step about copying dependencies

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

**Comment by olenagerasimova** [comment]
@genryxy thanks, added

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good

**Pull request merged**